tests/base/adapter.py

```diff
@@ -0,0 +1,43 @@
+
+import pytest
+
+from mock import MagicMock
+
+
+# We have to support customized adapter during the transition of accounts
+# between fxa and persona.
+
+
+@pytest.mark.django_db
+def test_adapter_base_get_connect_normal_auth_account(social_adapter0):
```

Comment:
`social_adapter0` is only used in this file, wouldn't it make sense to define it here? I'm not confident enough with pytest to know if that's even possible, but if it is, I think it would make it easier to understand those tests. I've found fixtures to be pretty obscure in our tests, so putting them closer to where they're being used would help imo.

Reply:
i think its a pretty common pattern in pytest to separate out the fixtures - but if its a blocker - i can move here - there's no technical reason why not

Reply:
Not a blocker, it simply was not obvious to me, and I actually made different comments because I misunderstood the fixture -- until I reached the end of the PR and saw the fixture itself, realizing my comments where completely off. 

So, I'm just sending that comment your way, in case you agree it would make it easier to understand, or if you have a better idea.

Reply:
i think my reluctance is that most of the fixtures are together now - so you might not see it if you needed to use this. That said - i want to make it as easy as possible to understand, so ill move it here for now, and we can move it to fixtures if it does need to be re-used

Reply:
updated